docs/docs/faq/index.md

```diff
@@ -56,9 +56,9 @@ class AcmeBankAccount implements AggregateRoot
 
 Let's re-use our `AcmeBankAccount` aggregate from the previous question. Now imagine after two years, you notice that
 you should have validated the IBAN data and that your existing aggregates now contain invalid data.
-Do not migrate existing events in this case! In event sourcing, events are immutable and must not be tempered with. 
+Do not migrate existing events in this case! In event sourcing, events are immutable and must not be tampered with. 
 
 One possible way of handling this business case would be to loop over all the existing aggregates, and fix them by recording
 another `IbanWasChanged` event, if you can fix them automatically. Or even better, a `InvalidIbanWasAutomaticallyFixed`
 event so that you can easily distinguish later on, what has been changed by e.g. a user or your own business logic.
- 
\ No newline at end of file
+ 
```

Comment:
GitHub adds a line break by default. Maybe we should just keep it for future contributions ;)